Can't create style-dictionary library with nx 17

### Current Behavior

Can't create a new style-dictionary library with nx 17

>  NX  Generating @nxkit/style-dictionary:library
 >  NX   Cannot find module '@nx/workspace/src/utilities/run-tasks-in-serial'
   Require stack:
   - C:\path\to\repo\node_modules\@nxkit\style-dictionary\src\generators\library\generator.js

### Expected Behaviour

Generator would work

### Github Repo

_No response_

### Steps to Reproduce

have a repo using at least nx 17.0.2 and try creating a library

### Nx Report

```shell
NX   Report complete - copy this into the issue template

   Node   : 20.9.0
   OS     : win32-x64
   npm    : 10.1.0

   nx (global)        : 17.0.2
   nx                 : 17.0.2
   @nx/js             : 17.0.2
   @nx/jest           : 17.0.2
   @nx/linter         : 17.0.2
   @nx/eslint         : 17.0.2
   @nx/workspace      : 17.0.2
   @nx/cypress        : 17.0.2
   @nx/devkit         : 17.0.2
   @nx/eslint-plugin  : 17.0.2
   @nx/nest           : 17.0.2
   @nx/next           : 17.0.2
   @nx/node           : 17.0.2
   @nx/react          : 17.0.2
   @nx/storybook      : 17.0.2
   @nrwl/tao          : 17.0.2
   @nx/web            : 17.0.2
   typescript         : 5.2.2
   ---------------------------------------
   Community plugins:
   @nx-tools/nx-container  : 5.1.0
   @nx-tools/nx-prisma     : 5.1.0
   @nxkit/style-dictionary : 3.0.2
   nx-stylelint            : 17.0.1
```


### Failure Logs

_No response_

### Additional Information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/packages/playwright/src/generators/init/generator.ts
+++ b/packages/playwright/src/generators/init/generator.ts
@@ -1,5 +1,4 @@
-import { formatFiles, GeneratorCallback, Tree } from '@nx/devkit';
-import { runTasksInSerial } from '@nx/workspace/src/utilities/run-tasks-in-serial';
+import { formatFiles, GeneratorCallback, Tree, runTasksInSerial} from '@nx/devkit';
 import { initGenerator as jsInitGenerator } from '@nx/js';
 import { addDependencies } from './lib/add-dependencies';
 import { installPlaywright } from './lib/install-playwright';

--- a/packages/playwright/src/generators/init/generator.ts
+++ b/packages/playwright/src/generators/init/generator.ts
@@ -1,4 +1,9 @@
-import { formatFiles, GeneratorCallback, Tree, runTasksInSerial} from '@nx/devkit';
+import {
+  formatFiles,
+  GeneratorCallback,
+  Tree,
+  runTasksInSerial,
+} from '@nx/devkit';
 import { initGenerator as jsInitGenerator } from '@nx/js';
 import { addDependencies } from './lib/add-dependencies';
 import { installPlaywright } from './lib/install-playwright';

--- a/packages/playwright/src/generators/preset/generator.ts
+++ b/packages/playwright/src/generators/preset/generator.ts
@@ -1,5 +1,4 @@
-import { names, Tree } from '@nx/devkit';
-import { runTasksInSerial } from '@nx/workspace/src/utilities/run-tasks-in-serial';
+import { names, Tree, runTasksInSerial } from '@nx/devkit';
 import projectGenerator from '../project/generator';
 import { PresetGeneratorSchema } from './schema';
 

--- a/packages/style-dictionary/src/generators/init/generator.ts
+++ b/packages/style-dictionary/src/generators/init/generator.ts
@@ -1,5 +1,9 @@
-import { formatFiles, GeneratorCallback, Tree } from '@nx/devkit';
-import { runTasksInSerial } from '@nx/workspace/src/utilities/run-tasks-in-serial';
+import {
+  formatFiles,
+  GeneratorCallback,
+  Tree,
+  runTasksInSerial,
+} from '@nx/devkit';
 import { initGenerator as jsInitGenerator } from '@nx/js';
 import { addDependencies } from './lib/add-dependencies';
 import { InitGeneratorSchema } from './schema';

--- a/packages/style-dictionary/src/generators/library/generator.ts
+++ b/packages/style-dictionary/src/generators/library/generator.ts
@@ -1,5 +1,9 @@
-import { formatFiles, joinPathFragments, Tree } from '@nx/devkit';
-import { runTasksInSerial } from '@nx/workspace/src/utilities/run-tasks-in-serial';
+import {
+  formatFiles,
+  joinPathFragments,
+  Tree,
+  runTasksInSerial,
+} from '@nx/devkit';
 import extensionGenerator from '../extension/generator';
 import { Extension } from '../extension/schema';
 import initGenerator from '../init/generator';

--- a/packages/style-dictionary/src/generators/preset/generator.ts
+++ b/packages/style-dictionary/src/generators/preset/generator.ts
@@ -1,5 +1,4 @@
-import { names, Tree } from '@nx/devkit';
-import { runTasksInSerial } from '@nx/workspace/src/utilities/run-tasks-in-serial';
+import { names, Tree, runTasksInSerial } from '@nx/devkit';
 import { libraryGenerator } from '../library/generator';
 import { PresetGeneratorSchema } from './schema';
 

